Xcodes/Backend/AppState.swift

```diff
@@ -299,7 +299,7 @@ class AppState: ObservableObject {
                     .mapError { $0 as Error }
             }
             .flatMap { [unowned self] in
-                self.install(.version(availableXcode), downloader: .aria2)
+                self.install(.version(availableXcode), downloader: Downloader(rawValue: UserDefaults.standard.string(forKey: "downloader") ?? "urlSession") ?? .urlSession)
```

Comment:
I think we should fall back to .aria2 to match the existing behaviour and the code in AdvancedPreferencePane.

```suggestion
                self.install(.version(availableXcode), downloader: Downloader(rawValue: UserDefaults.standard.string(forKey: "downloader") ?? "aria2") ?? .aria2)
```